### ISSUE TITLE
Vue3 form helper with 'remember' throws error if data contains array

### ISSUE THREAD

**Issue by bradgenereux** [inertiajs/inertia#552]
### Versions:

- `@inertiajs/inertia` version: 0.8.5
- `@inertiajs/inertia-vue3` version: 0.3.5

### Describe the problem:

Using 'remember' with the form helper throws an error if the data contains an array.

```DOMException: Failed to execute 'replaceState' on 'History': [object Object] could not be cloned.```

### Steps to reproduce:

```js
export default {
  remember: 'form',
  data() {
    return {
      form: this.$inertia.form({
        name: '',
        items: [],
      }),
    }
  }
}
```
It works without the form helper. And it works in Vue2 with and without the form helper.

**Comment by ortzinator** [comment]
I just ran into the same problem

**Comment by reinink** [comment]
What kind of data are you putting into the array?

**Comment by reinink** [comment]
Scratch that, I am able to reproduce this. 👍

**Comment by reinink** [comment]
Figured this out, and a fix in coming in #575. 🙌

**Comment by ortzinator** [comment]
I updated to @inertiajs/inertia-vue3@0.3.6 and I get the same error.

https://gist.github.com/ortzinator/66d12948b170d7da0f79720b4e12c09d

**Comment by yordi024** [comment]
> I updated to @inertiajs/inertia-vue3@0.3.6 and I get the same error.
> 
> https://gist.github.com/ortzinator/66d12948b170d7da0f79720b4e12c09d

I am getting the same error

**Comment by bradgenereux** [comment]
@ortzinator @yordi024 
Seems to be a problem with prop data. I've opened a new issue here #582

**Comment by reinink** [comment]
Fixed (again) in #587 👍
